mirascope/chat/utils.py

```diff
@@ -55,10 +82,10 @@ def patch_openai_kwargs(
 
 
 def convert_function_to_openai_tool(fn: Callable) -> Type[OpenAITool]:
-    """Constructs and `OpenAITool` type from the given function.
+    """Constructs an `OpenAITool` type from the given function.
 
-    If parameters are not defined in the Args section, then the description will simply
-    be the name of the parameter.
+    This method assumes all function parameters are properly documented in identical
```

Comment:
Don't say `assume` here. We don't assume, we expect. We throw errors if not so there shouldn't be silent failures if it doesn't conform.